src/cmd/manage_nsfs.js

```diff
@@ -352,6 +376,17 @@ async function update_bucket(data) {
     write_stdout_response(ManageCLIResponse.BucketUpdated, data);
 }
 
+/** verify_update_bucket will check if the bucket_owner has an account
+ * in case it does not find one, it would throw an error
+ * @param {string} bucket_owner
+ */
+async function verify_update_bucket(bucket_owner) {
```

Comment:
there is no need to have a different function for update, use the same function as for create